SBOMDocumentTypes.md

```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SBOMDocumentTypes
+
+## Summary
+
+Provides a set of values to be used to describe the common types of SBOMs that tools may create.
+
+## Description
+
+This field provides a list with agreed on definitions of SBOM types, as defined in [Types of Software Bill of Material (SBOM) Documents](https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf), published on April 21, 2023. 
+This field is a reasonable estimate of the most likely type of an SBOM from the producer and consumer perspective from which both parties can draw conclusions
```

Comment:
The description "reasonable estimate of the most likely type of an SBOM from the **producer and consumer perspective**"  is inconsistent with the `sbomType` field below which says: "This field is a reasonable estimation of the type of SBOM created from a **creator perspective**"